int8 enable
int8 run
int8 gui_connection_requierd
float64 max_speed
time time_frame
